PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,34 @@
-High level description including:
- - What does this change enable
- - What bugs does this change fix
- - High-Level how things changed
+___
+**What is the review process and when will my changes land?**
 
-# Issues
+All PR's require 2 approvals using github's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).
 
-[DCOS-<number>](https://dcosjira.atlassian.net/browse/DCOS-<number>)
-...
+Reviewers should be:
+* Developers who understand the code being modified.
+* Developers responsible for code that interacts with or depends on the code being modified.
 
-# Checklist
+It is best to proactively ask for 2 reviews by mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the next train. Trains usually depart the station within 1 business day of landing.
 
- - [ ] Included a test which will fail if code is reverted but test is not
- - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)
+**PLEASE FILL IN THE TEMPLATE BELOW** / **DO NOT REMOVE ANY SECTIONS BELOW THIS LINE**
+___
+## High Level Description
 
-If this is a component/package update, include
+What features does this change enable? What bugs does this change fix? High-Level description of how things changed.
 
- - [ ] Change Log from last: <link>
- - [ ] Test Results: <link>
- - [ ] Code Coverage: <link>
+## Related Issues
 
+  - [DCOS-<number>](https://dcosjira.atlassian.net/browse/DCOS-<number>)
+
+## Checklist for all PR's
+
+  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
+  - [ ] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
+  - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)
+
+## Checklist for component/package updates:
+
+If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to above please also include: 
+
+  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/mesos_state/compare/3db593a81d3caa18b87468754918ca3771e8cace...7fa7e6edb4d2eecc0ee12a1f109accddf1978953)
```

Comment:
Could you please use an example repo that we actually include in dcos? (mesos_state is deprecated/removed)
Example: 
https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae